app/scripts/components/boilerplate/divider.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+require('stylesheets/boilerplate/divider');
+
+const Divider = React.createClass({
+  render() {
+    return (
+      <div className="layoutdivider"></div>
```

Comment:
Keep a consistent name for the filename, component name, and classname, so this should just be `className="divider"`.